frontend/src/pages/Route/transform.ts

```diff
@@ -152,10 +152,6 @@ export const transformRouteData = (data: RouteModule.Body) => {
 
   const { plugins, script } = data;
 
-  if (plugins && plugins.prometheus) {
-    plugins.prometheus = { enabled: true };
```

Comment:
If we remove this line, and just enabled this plugin, then how about this plugin's status when we use Edit mode? Is it still enabled?

Reply:
> If we remove this line, and just enabled this plugin, then how about this plugin's status when we use Edit mode? Is it still enabled?

When we in Edit mode, API will return `plugins:{promethues:{}}`if we enabled it before and it works as expected.

![image](https://user-images.githubusercontent.com/31329157/97936301-10521400-1db6-11eb-9793-40611eeb9afa.png)